pg_metadata/install/sql/upgrade/upgrade_to_0.2.2.sql

```diff
@@ -26,6 +26,11 @@ CREATE SEQUENCE pgmetadata.theme_id_seq
 ALTER SEQUENCE pgmetadata.theme_id_seq OWNED BY pgmetadata.theme.id;
 
 
+-- theme theme_pkey
+ALTER TABLE ONLY pgmetadata.theme
```

Comment:
This version has already been released.
You need a new migration file, otherwise, this change will never be applied on databases where PgMetadata is already installed.